l3experimental/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ this project uses date-based 'snapshot' version identifiers.
 ### Added
 - `\draw_path_replace_bb:`
 
+### Changed
+- `l3opacity` is sped up a bit
```

Comment:
Do we need to document in the backend ChangeLog? I assume we are relying on a change-in-behaviour in teh frontend so should do.